data/remnant/remnant.txt

```diff
@@ -276,6 +276,498 @@ phrase "remnant"
+		"slow"
+	word
+		" "
+	word
+		"the enemy's thrusters"
+		"your target's steering"
+		"any engine"
+	word
+		" "
+	word
+		"and prevent them from escaping"
+		"that anyone will surely beg you to stop"
+	word
+		"."
+		"!"
+
+phrase "remnant thrasher cannon"
+	word
+		"a dozen of"
+		"several"
+		"some"
+	word
+		" "
+	word
+		"thrasher cannons"
+		"thrasher guns"
+		"thrasher turrets"
+	word
+		"."
+	word
+		" "
+	word
+		"Compared to"
+		"When placed beside"
+	word
+		" "
+	word
+		"your energy blasters,"
+	word
+		" "
+	word
+		"they are more efficient in terms of heat"
+		"you can deal more damage with our cannons"
+		"they may be consuming more energy, but they have better heat and damage"
+		"our cannons may be more expensive, but they have better damage"
+		"they are larger, but they can deal more damage and release less heat"
+		"they can be a force to reckoned with"
+	word
+		"."
+		"!"
+
+phrase "remnant emp torpedo bay"
+	word
+		"an EMP Torpedo Bay."
+	word
+		" "
+	word
+		"With a single shot, it"
+		"Firing one"
+		"One torpedo"
+	word
+		" "
+	word
+		"will"
+		"can"
+	word
+		" "
+	word
+		"disrupt"
+		"disable"
+		"deplete"
+	word
+		" "
+	word
+		"the enemy's systems"
+		"your target's batteries"
+		"your enemy's shields"
+		"the target's engines"
+	word
+		"."
+		"!"
+
+phrase "remnant salvage scanner"
+	word
+		"one or two"
+		"a dozen"
+		"some"
+	word
+		" "
+	word
+		"salvage scanners."
+	word
+		" "
+	word
+		"They are a combination of an outfit scanner and a tactical one"
+		"Compared to your separated versions, our special scanner is more powerful"
+		"They have a longer range than your outfit scanner"
+		"Those babies scan fast"
+		"They are a must for your plundering needs"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant"
+	word
+		"Have you heard"
+		"Do you know"
+		"Are you familiar"
+	word
+		" "
+	word
+		"about"
+		"regarding"
+	word
+		" "
+	word
+		"the"
+		"those"
+	word
+		" "
+	word
+		"Quarg"
+		"Pug"
+		"Hai"
+		"Drak"
+		"Archons"
+		"insects far west of our space"
+	word
+		"?"
+	word
+		" "
+	phrase
+		"friendly remnant (aliens.!)"
+		"friendly remnant (aliens?)"
+
+phrase "friendly remnant (aliens.!)"
+	word
+		"They look strong"
+		"They look weak"
+		"Maybe we should contact them"
+		"I think they are dangerous"
+		"I think they are good"
+		"I think we can trust them"
+		"I want to plunder their tech"
+		"I want their ships"
+		"I have seen them"
+		"I haven't seen them yet"
+		"I wonder if the prefects already knew about them"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant (aliens?)"
+	word
+		"Are they strong"
+		"Are they weak"
+		"Are they friendly"
+		"Are they hostile"
+		"How strong are they"
+		"What are the odds of capturing their ships"
+		"Did your government met them"
+		"Is it worth plundering them"
+	word
+		"?"
+
+phrase "friendly remnant"
+	word
+		"Be careful"
+		"Stay alert"
+		"Be vigilant"
+	word
+		" "
+	word
+		"about"
+		"aboot"
+	word
+		" "
+	word
+		"those"
```

Comment:
Those does not work with "Archon in Nenia".
I suggest maybe putting the or those before the individua phrases there.